From d944a74615424ab65fc196ec580995afff7b4cd6 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Thu, 8 Dec 2016 09:18:32 +0800
Subject: [PATCH 1326/5242] MLK-13557 imx6sll: add fsl,wdog_b and pinmux
 settings

commit  874185c733f114c67719f68998e5afe86564d56e from
https://source.codeaurora.org/external/imx/linux-imx.git

Add fsl,wdog_b and pinmux settings.
We need to use wdog_b to trigger pmic reset, when we trigger
wdog_b reset, we should not trigger wdog_reset_b_deb.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
(cherry picked from commit 0400ecd8fb3a3772dfb3a68c55960419a63f1650)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll-evk.dts |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sll-evk.dts b/arch/arm/boot/dts/imx6sll-evk.dts
index b4af007..68d56a7 100644
--- a/arch/arm/boot/dts/imx6sll-evk.dts
+++ b/arch/arm/boot/dts/imx6sll-evk.dts
@@ -671,6 +671,12 @@
 				MX6SLL_PAD_PWM1__PWM1_OUT   0x110b0
 			>;
 		};
+
+		pinctrl_wdog1: wdog1grp	{
+			fsl,pins = <
+				MX6SLL_PAD_WDOG_B__WDOG1_B   0x170b0
+			>;
+		};
 	};
 };
 
@@ -799,3 +805,9 @@
 &ssi2 {
 	status = "okay";
 };
+
+&wdog1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_wdog1>;
+	fsl,ext-reset-output;
+};
-- 
1.7.9.5

